# Request
int16 id
---
# Response
bool idgrab
bool success
